### PR TITLE
Mentions/message

### PR DESCRIPTION
**Pull request by sjaakluthart**

### CODE REVIEW

File: src/message/index.jsx [hunk 1]
```diff
@@ -85,6 +85,12 @@ class Message extends Component {
     locale: PropTypes.instanceOf(Object),
     /** Show a separator above the message */
     separator: PropTypes.node,
+    /**
+     * Mention users, should be an array of usernames
+     *
+     * Text matching '@username' will be highlighted
+     */
+    mentions: PropTypes.arrayOf(String),
```
Comment (IanCStewart) [inline src/message/index.jsx]: I don't really like this naming. This array seems more like possibleMentions then the mentions it self.

Comment (sjaakluthart) [inline src/message/index.jsx]: The mentions array is an array of usernames that are mentioned in a certain message, therefore `mentions` seems fine to me.

Comment (IanCStewart) [inline src/message/index.jsx]: Ah that seems right then indeed. Maybe make the description a bit more concrete then.

**Comment by IanCStewart** [comment]
@larstadema please review `parse` part

**Pull request merged**